include/ultra64/os.h

```diff
@@ -0,0 +1,26 @@
+#ifndef ULTRA64_OS_H
+#define ULTRA64_OS_H
+
+#include "os_ai.h"
+#include "os_cache.h"
+#include "os_cont.h"
+#include "os_convert.h"
+#include "os_exception.h"
+#include "os_flash.h"
+#include "os_host.h"
+#include "os_internal_error.h"
+#include "os_internal_reg.h"
+#include "os_internal_si.h"
+#include "os_internal.h"
+#include "os_libc.h"
+#include "os_message.h"
+#include "os_pfs.h"
+#include "os_pi.h"
+#include "os_reg.h"
+#include "os_system.h"
+#include "os_thread.h"
+#include "os_time.h"
+#include "os_tlb.h"
+#include "os_voice.h"
+
+#endif
```

Comment:
Is the intent with `os.h` to include `os.h` whenever you want to use any of these files or could you still include individual files? For example, in audio, would it be preferred to include `#include "os.h"` or could I just `#include "os_ai.h"` as needed?

Reply:
Actually that's a good question. I'm not completely sure. I would say only include one file if you only need one of those files and use `os.h` if you need more than one